src/app/help/en/faq.md

```diff
@@ -1,10 +1,8 @@
 # Steemit FAQ 
 
-<a class="anchor" name="Table_of_Contents"></a>
-## Table of Contents
+## Table of Contents <a class="anchor" name="Table_of_Contents"></a>
```

Comment:
`Table of Contents` should be within the < a > tag. Actual a tag should use `id=` not `name=`

.anchor will need new styling to remove _cursor is pointer_ appearance and color.

This applies to all _anchor_ a tags in this PR.

Reply:
If I put the `Table of Contents` text inside of the `<a...` portion of the tag, the text no longer shows up on the UI.

![image](https://user-images.githubusercontent.com/20735105/33409597-e569ce50-d541-11e7-9f07-6c42b62d287f.png)


![image](https://user-images.githubusercontent.com/20735105/33409578-ce4a3020-d541-11e7-96ed-7fc47f6b5641.png)